mssql-ballerina/Ballerina.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+org = "ballerinax"
+name = "mssql"
+version = "0.1.0"
+
+[[platform.java11.dependency]]
+path = "../mssql-native/build/libs/mssql-native-0.1.0-SNAPSHOT.jar"
+groupId = "org.ballerinalang"
+artifactId = "mssql-native"
+version = "0.1.0-SNAPSHOT"
+
+[[platform.java11.dependency]]
+path = "/home/rajinthan/Documents/GitHub/module-ballerinax-mssql/mssql-ballerina/build/target/jballerina-tools-2.0.0-alpha6-SNAPSHOT/repo/bala/ballerina/sql/0.6.0-alpha5/java11/platform/java11/sql-native-0.6.0-alpha5-SNAPSHOT.jar"
```

Comment:
DO NOT add a machine-specific path. Use system variables to update and revert these. Same as version

Reply:
Fixed with [`9dfb4e8`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/98fb4e854661be330b7cf90f81bf53ec0b3f522a)